src/adapter/src/catalog/builtin.rs

```diff
@@ -46,7 +46,7 @@ use crate::catalog::storage::{MZ_SUPPORT_ROLE_ID, MZ_SYSTEM_ROLE_ID};
 use crate::catalog::DEFAULT_CLUSTER_REPLICA_NAME;
 use crate::rbac;
 
-pub static BUILTIN_PREFIXES: Lazy<Vec<&str>> = Lazy::new(|| vec!["mz_", "pg_", "external_"]);
+pub const BUILTIN_PREFIXES: &[&str] = &["mz_", "pg_", "external_"];
```

Comment:
The `static` to `const` changes are probably unnecessary. I originally added them so that I could add some const asserts, but that ended up not being possible. I ended up like the `const` changes, though so I kept them.